### PR TITLE
fix: Invalid SteamID #5

### PR DESCRIPTION
**Pull request by Rushaway**

### CODE REVIEW

File: addons/sourcemod/scripting/vip/UTIL.sp [hunk 1]
```diff
@@ -299,7 +299,7 @@ void UTIL_ADD_VIP_PLAYER(int iAdmin = 0,
 	{
 		GetClientName(iTarget, SZF(szQuery));
 		g_hDatabase.Escape(szQuery, SZF(szName));
-		iAccountID = GetSteamAccountID(iTarget);
+		iAccountID = GetSteamAccountID(iTarget, false);
```
Comment (maxime1907) [inline addons/sourcemod/scripting/vip/UTIL.sp:302]: also check the return value if its 0, throw an error:
- https://sourcemod.dev/#/clients/function.GetSteamAccountID

[the remark this answers is not in the document]
Comment (Rushaway) [reply addons/sourcemod/scripting/vip/UTIL.sp:302]: If a player is 'in Steamid pending' and get added as VIP, the iAccountID will return 0 and get inserted in the DB.

[the remark this answers is not in the document]
Comment (maxime1907) [reply addons/sourcemod/scripting/vip/UTIL.sp:302]: okay so now with the check you added it should be fixed so no need to pass false here

**Comment by maxime1907** [comment]
merging it here for now but could you open the same PR upstream?

**Pull request merged**